wcivf/apps/results/templates/results/results_list.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block content %}
+<section class="card">
+  <h1>Latest election results</h1>
+  {% for election_dict in elections %}
+    <h2>{{ election_dict.election }}</h2>
+    {% for result in election_dict.results %}
+      {% for person_post in result.person_posts.all %}
```

Comment:
Ah - I think this template logic needs another look - just tried it against the live Atom feed and it's (i) printing results for non-elected candidates and (ii) iterating more than it should be. I see 16 cards for 4 candidates in Woodville ward, of whom only 1 was elected.